### ISSUE TITLE
Amend uses of get_terms() arguments.

### ISSUE THREAD

**Issue by tw2113** [WebDevStudios/custom-post-type-ui#563]
Dependent on #562 

Old:

```
$terms = get_terms( 'post_tag', array(
    'hide_empty' => false,
) );
```

New:
```
Since 4.5.0, taxonomies should be passed via the ‘taxonomy’ argument in the $args array:

$terms = get_terms( array(
    'taxonomy' => 'post_tag',
    'hide_empty' => false,
) );
```

Old way still works, but we should not use legacy versions whenever possible.

**Comment by tw2113** [comment]
Fixed in 2936e80

**Comment by tw2113** [comment]
Fixed in 2936e80
